core/modules/exact_match.cc

```diff
@@ -43,6 +43,12 @@ static inline int is_valid_gate(gate_idx_t gate) {
 }
 
 const Commands ExactMatch::cmds = {
+    {"GetArg", "EmptyArg", MODULE_CMD_FUNC(&ExactMatch::GetArg),
+     Command::THREAD_UNSAFE},
+    {"GetMut", "EmptyArg", MODULE_CMD_FUNC(&ExactMatch::GetMut),
+     Command::THREAD_UNSAFE},
```

Comment:
Can we mark these as thread safe?

Reply:
Are we guaranteed one thread at a time executing commands?

Reply:
Yup (https://github.com/NetSys/bess/blob/e11f78916cbe4e4139ad14ea5f31e83ba72b88e2/core/bessctl.cc#L1640)